datahub-web-react/src/app/entity/shared/EntityDropdown/AddDeprecationDetailsModal.tsx

```diff
@@ -6,7 +6,7 @@ type Props = {
     urn: string;
     visible: boolean;
     onClose: () => void;
-    refetch?: () => Promise<any>;
```

Comment:
why make this change? isn't this just less explicit typing?

Reply:
not really - it doens't have any real importance inside the modal at all (and was causing some issues when i wanted to pass a function that doesn't explicitly return a Promise